qa/tasks/ceph_test_case.py

```diff
@@ -36,9 +136,23 @@ class CephTestCase(unittest.TestCase):
     # their special needs.  If not met, tests will be skipped.
     REQUIRE_MEMSTORE = False
 
+    def _init_mon_manager(self):
+        # if vstart_runner.py has invoked this code
+        if 'Local' in str(type(self.ceph_cluster)):
+            from tasks.vstart_runner import LocalCephManager
+            self.mon_manager = LocalCephManager(ctx=self.ctx)
```

Comment:
@rishabh-d-dave 

```
2024-03-13 19:29:15,638.638 INFO:__main__:----------------------------------------------------------------------
2024-03-13 19:29:15,639.639 INFO:__main__:Traceback (most recent call last):
2024-03-13 19:29:15,639.639 INFO:__main__:  File "/home/jenkins-build/build/workspace/ceph-api/qa/tasks/mgr/test_dashboard.py", line 16, in setUp
2024-03-13 19:29:15,639.639 INFO:__main__:    super(TestDashboard, self).setUp()
2024-03-13 19:29:15,639.639 INFO:__main__:  File "/home/jenkins-build/build/workspace/ceph-api/qa/tasks/ceph_test_case.py", line 153, in setUp
2024-03-13 19:29:15,639.639 INFO:__main__:    self._init_mon_manager()
2024-03-13 19:29:15,639.639 INFO:__main__:  File "/home/jenkins-build/build/workspace/ceph-api/qa/tasks/ceph_test_case.py", line 143, in _init_mon_manager
2024-03-13 19:29:15,639.639 INFO:__main__:    self.mon_manager = LocalCephManager(ctx=self.ctx)
2024-03-13 19:29:15,639.639 INFO:__main__:TypeError: LocalCephManager.__init__() got an unexpected keyword argument 'ctx'
```

Reply:
Recent push should fix it, I'll keep an eye.

Reply:
Fix worked, API tests passed this time - https://jenkins.ceph.com/job/ceph-api/70575/